src/Data/MediaType/Common.purs

```diff
@@ -40,3 +40,6 @@ textPlain = MediaType "text/plain"
 
 textXML :: MediaType
 textXML = MediaType "text/xml"
+
+textCss :: MediaType
```

Comment:
Specifically I mean changing this to textCSS

Reply:
tnx for quick response, done)